plugins/woocommerce-blocks/packages/interactivity-components/dropdown/style.scss

```diff
@@ -1,145 +1,90 @@
-@import "../../../assets/js/blocks/shared/styles/style";
-
 .wc-interactivity-dropdown {
```

Comment:
![image](https://github.com/woocommerce/woocommerce/assets/5423135/5ab379a8-ce19-4e85-8219-8742fd2d98e6)

Reply:
Good spotting! I fixed that here https://github.com/woocommerce/woocommerce/pull/43183/commits/e414cd0300130c198aa10723b9ca8aaccec70827

Preview:

<img width="162" alt="Screenshot 2024-01-02 at 11 04 32 AM" src="https://github.com/woocommerce/woocommerce/assets/1281828/ea9843c0-a3a0-4ee5-8b01-c3cff6417992">